### DISCUSSION TITLE
Taskbar Labels for Windows 11 1.2 - labels no longer show correctly

### DISCUSSION THREAD

**Discussion by AJPopple** [ramensoftware/windhawk-mods#400] [General]
Just upgraded "Taskbar Labels for Windows 11" to v1.2 and the labels no longer show correctly in the taskbar buttons.
For a brief second after I "Save settings" the labels are correct, but then they change to something more generic.

ie, File explorer becomes "File Explorer..." or "Windows...", LibreOffice Calc becomes "soffice.bin",  Firefox is "Firefox" instead of the page title, etc.

**Comment by MamiyaOtaru** [comment]
if you turn on Windows' built in tasbkar labels the correct labels come back.  Why use Windhawk Taskbar Labels then?  Being able to define the taskbar button width instead of having them automatically shrink but never grow like with Windows' implementation I guess :)

**Comment by m417z** [comment]
Should be fixed in v1.2.1, see https://github.com/ramensoftware/windhawk-mods/issues/335#issuecomment-1832312172.

**Reply by AJPopple** [reply]
Spot on, cheers!

**Comment by MamiyaOtaru** [comment]
the minimum width setting brings this to feature parity with explorerpatcher's taskbar imho.  It's really solid right now.
